### PR TITLE
Add enableStatus

### LINKED ISSUE #12: Integrate reactive-lib health and status checks
sbt integrates health and status checks.  maven should replicate this

### PR DESCRIPTION
**Pull request by eed3si9n**
Fixes #12

### CODE REVIEW

File: src/main/java/com/lightbend/rp/BasicApp.java [hunk 1]
```diff
@@ -67,7 +67,11 @@ public void apply(MavenProject project) {
             labels.add("modules.service-discovery.enabled", "true");
         if(settings.enableAkkaClusterBootstrap) {
             labels.add("modules.akka-cluster-bootstrap.enabled", "true");
+            labels.add("modules.status.enabled", "true");
```
Comment (eed3si9n) [inline src/main/java/com/lightbend/rp/BasicApp.java]: Is this all I need to do?

Comment (mitkus) [reply src/main/java/com/lightbend/rp/BasicApp.java]: That's a part of it, yes. For status and health urls to work, they also need to be enabled in `application.conf`, like this: https://github.com/lightbend/reactive-lib/blob/master/status/src/main/resources/rp-tooling.conf. `sbt-reactive-app` does this by having a merging feature - `application.conf` gets merged with `rp-tooling.conf` provided by libraries, at build time.

Comment (mitkus) [reply src/main/java/com/lightbend/rp/BasicApp.java]: I wrote a comment about my experience with merging configs here: https://github.com/lightbend/reactive-app-maven-plugin/issues/12

File: src/main/java/com/lightbend/rp/MergeConfigMojo.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        Xpp3Dom pluginConf = (Xpp3Dom)getThisPlugin().getConfiguration();
+        Settings settings = new Settings();
+        settings.read(pluginConf);
+        if(settings.enableAkkaClusterBootstrap
+            || settings.enableStatus) {
+            log.info("Executing merge-config");
+            try {
+                mergeConfigFiles(log);
+            } catch (Throwable e) {
+                throw new MojoExecutionException(e.getMessage(), e);
```
Comment (dwijnand) [inline src/main/java/com/lightbend/rp/MergeConfigMojo.java]: Don't use e.getMessage, that's already in e. Either provide provide context with a message (e.g "Failed to something merge config something"), or don't set any message at all.

**Pull request merged**